mono/eglib/test/spawn.c

```diff
@@ -22,6 +22,7 @@
 static RESULT
 test_spawn_sync (void)
 {
+#ifndef G_OS_WIN32
```

Comment:
Should be `#if HAVE_G_SPAWN` or `HAS_G_SPAWN`, but really, probably the same thing.